pkgs/development/python-modules/python-snappy/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, buildPythonPackage, fetchPypi, python, snappy, cffi }:
+
+buildPythonPackage rec {
+  pname = "python-snappy";
+  version = "0.5.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0jnp4xxaif585l76nqf3fviva4iclgdd51sksl342g880qzq0zwa";
+  };
+
+  propagatedBuildInputs = [ snappy cffi ];
+
+  checkPhase = ''
+    export PYTHONPATH=$out/${python.sitePackages}:$PYTHONPATH
+    HOME=$(mktemp -d) ${python.interpreter} setup.py test
```

Comment:
`${python.interpreter} setup.py test` is run by default, so the other contents could be put in the `preCheck` instead.